intents/data/obj.json

```diff
@@ -0,0 +1 @@
+{"intent":"topk","table":[["idx","flno","origin","destination","distance","departure_date","arrival_date","price","aid"],[0,99,"Los Angeles","Washington D.C.",2308,"2005-04-04T18:30:00.000Z","2005-04-04T18:30:00.000Z",235.98,1],[1,13,"Los Angeles","Chicago",1749,"2005-04-04T18:30:00.000Z","2005-04-04T18:30:00.000Z",220.98,3],[2,346,"Los Angeles","Dallas",1251,"2005-04-04T18:30:00.000Z","2005-04-04T18:30:00.000Z",182,2],[3,387,"Los Angeles","Boston",2606,"2005-04-04T18:30:00.000Z","2005-04-04T18:30:00.000Z",261.56,6],[4,7,"Los Angeles","Sydney",7487,"2005-04-04T18:30:00.000Z","2005-04-04T18:30:00.000Z",278.56,3],[5,2,"Los Angeles","Tokyo",5478,"2005-04-04T18:30:00.000Z","2005-04-04T18:30:00.000Z",780.99,9],[6,33,"Los Angeles","Honolulu",2551,"2005-04-04T18:30:00.000Z","2005-04-04T18:30:00.000Z",375.23,7],[7,34,"Los Angeles","Honolulu",2551,"2005-04-04T18:30:00.000Z","2005-04-04T18:30:00.000Z",425.98,5],[8,76,"Chicago","Los Angeles",1749,"2005-04-04T18:30:00.000Z","2005-04-04T18:30:00.000Z",220.98,9],[9,68,"Chicago","New York",802,"2005-04-04T18:30:00.000Z","2005-04-04T18:30:00.000Z",202.45,10],[10,99,"Los Angeles","Washington D.C.",2308,"2005-04-12T18:30:00.000Z","2005-04-12T18:30:00.000Z",235.98,1],[11,13,"Los Angeles","Chicago",1749,"2005-04-12T18:30:00.000Z","2005-04-12T18:30:00.000Z",220.98,3],[12,346,"Los Angeles","Dallas",1251,"2005-04-12T18:30:00.000Z","2005-04-12T18:30:00.000Z",182,2],[13,387,"Los Angeles","Boston",2606,"2005-04-12T18:30:00.000Z","2005-04-12T18:30:00.000Z",261.56,6],[14,7,"Los Angeles","Sydney",7487,"2005-04-12T18:30:00.000Z","2005-04-12T18:30:00.000Z",278.56,3],[15,2,"Los Angeles","Tokyo",5478,"2005-04-12T18:30:00.000Z","2005-04-12T18:30:00.000Z",780.99,9],[16,33,"Los Angeles","Honolulu",2551,"2005-04-12T18:30:00.000Z","2005-04-12T18:30:00.000Z",375.23,7],[17,34,"Los Angeles","Honolulu",2551,"2005-04-12T18:30:00.000Z","2005-04-12T18:30:00.000Z",425.98,5],[18,76,"Chicago","Los Angeles",1749,"2005-04-12T18:30:00.000Z","2005-04-12T18:30:00.000Z",220.98,9],[19,68,"Chicago","New York",802,"2005-04-12T18:30:00.000Z","2005-04-11T18:30:00.000Z",202.45,10]],"rowRange":{"header":1,"rowStart":2,"rowEnd":21},"metric":"price","dimensions":["origin"],"summaryOperator":"Mean","isAsc":false,"k":5,"slices":"null","dateRange":"null","timeGranularity":"null"}
```

Comment:
Do add newline at end of file.

Reply:
Was not aware of placing a newline at end of a .json file. But will add it.